addons/aircraft/XEH_PREP.hpp

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
There is a basic template in extras. Whitespace at the top is  redundant, you should never read decompiled code unless debugging tools or something.